docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,271 @@
+# Getting Started With Soci Snapshotter
+This document is meant to be a guide to walk through an example of using the
+soci-snapshotter.  To get everything running we will need to do a few things : 
+- prerequistes
+- start a local oras registry
```

Comment:
Since ORAS does not have its own registry implementation, it might be better to call this a "references-enabled OCI registry"? I know that's a mouthful, but I don't think we want to cause confusion that there is something called the "ORAS registry." Given the initial OCI spec changes for "references" were merged yesterday, very soon there could be other options here. We are just giving people an easy starting point with the Docker distribution (registry) project fork in ORAS which already has this feature implemented.

Reply:
Rather than using vague language that captures both ORAS and Proposal E, how about leaving the explicit ORAS references, but add a note saying that the project will be replacing ORAS support with Reference Types support "real soon now", and that these instructions will be updated when that happens.

You can even link to the the issue for same: https://github.com/awslabs/soci-snapshotter/issues/42